mixes/containers/mix.js

```diff
@@ -7,25 +7,32 @@ const { loadMix, saveMix } = require('../actions')
 class MixContainer extends React.Component {
   render () {
     const { mix, isLoading, isSaving, error, saveMix } = this.props
+    if (!mix) { return null }
```

Comment:
wdyt of this? should views be defensive? i was running into race conditions with calling `navigateToMix` and `createMeta` both in the same batch from `createMix`.

i think the trend there is i'm finding it tough to sequence these sorts of things with batching, eg only calling end when they've all ended... since `Effects.batch` operates in parallel right? and each job doesnt wait for the other jobs' subjobs, counter to how promises can be chained.

whats the correct pattern there? should i be only batching together actions that can be perfectly parallelized, and forcing others to come after a batch?